EmailNotification/__tests__/handler.test.ts

```diff
@@ -143,52 +143,38 @@ const mockRetrieveProcessingMessageData = jest.fn().mockImplementation(() =>
 );
 
 describe("getEmailNotificationActivityHandler", () => {
-  test.each([
-    {
-      betaList: [],
-      ff: FeatureFlagEnum.ALL,
-      expectedHtmlFormTemplate: true
-    }
-  ])(
-    "GIVEN an EmailNotification.handeler with FF: $ff and beta_list: $betaList WHEN the handler run with input $input THEN should return SUCCESS with the right html email body",
-    async ({ betaList, ff, expectedHtmlFormTemplate }) => {
-      const mockSendMail = jest
-        .spyOn(mail, "sendMail")
-        .mockReturnValueOnce(TE.of("SUCCESS"));
-
-      const GetEmailNotificationActivityHandler = getEmailNotificationHandler(
-        lMailerTransporterMock,
-        notificationModelMock,
-        mockRetrieveProcessingMessageData,
-        defaultNotificationParams,
-        ({ BETA_USERS: betaList, FF_TEMPLATE_EMAIL: ff } as unknown) as IConfig
-      );
+  test("GIVEN an EmailNotification.handler WHEN the handler run  THEN should return SUCCESS with the right html email body", async () => {
+    const mockSendMail = jest
+      .spyOn(mail, "sendMail")
+      .mockReturnValueOnce(TE.of("SUCCESS"));
 
-      const result = await GetEmailNotificationActivityHandler(
-        mockContext,
-        JSON.stringify(input)
-      );
+    const GetEmailNotificationActivityHandler = getEmailNotificationHandler(
+      lMailerTransporterMock,
+      notificationModelMock,
+      mockRetrieveProcessingMessageData,
+      defaultNotificationParams
+    );
 
-      const expectedHtml = expectedHtmlFormTemplate
-        ? messagetemplate.apply(
-            aMessageContent.subject,
-            (await markdownToHtml.process(aMessageContent.markdown)).toString(),
-            aSenderMetadata
-          )
-        : await generateDocumentHtml(
-            aMessageContent.subject,
-            aMessageContent.markdown,
-            aSenderMetadata
-          );
-
-      expect(mockSendMail).toBeCalledWith(
-        expect.anything(),
-        expect.objectContaining({ html: expectedHtml })
-      );
+    const result = await GetEmailNotificationActivityHandler(
+      mockContext,
+      JSON.stringify(input)
+    );
 
-      expect(result.kind).toBe("SUCCESS");
-    }
-  );
+    const expectedHtml = messagetemplate.apply(
+      aMessageContent.subject,
+      (
+        await markdownToHtml.process(prepareBody(aMessageContent.markdown))
+      ).toString(),
+      aSenderMetadata
+    );
+
+    expect(mockSendMail).toBeCalledWith(
+      expect.anything(),
```

Comment:
```suggestion
      lMailerTransporterMock
```

Reply:
Thank you for the suggestion, did it in [c416cb2](https://github.com/pagopa/io-functions-services/pull/277/commits/c416cb2df2c78b559a44e0b5f4cd78b368e6af7c)